src/stratigraphy/util/find_description.py

```diff
@@ -44,7 +58,17 @@ def get_description_blocks(
             ):
                 blocks.append(TextBlock(current_block_lines, is_terminated_by_line=True))
                 current_block_lines = []
-
+            elif _block_separated_by_lefthandside_line_segment(
```

Comment:
The check for both, lines as well as the lefthandside lines, leads to some trouble in the recognition.

Sometimes, a block is split by a lefthandside line. The resulting block consists of a single textline, which may be short. Therefore, the new block is small in its width. The next lefthandside line may now be detected as a "full line". The block is split correctly, but then later, we do not apply the splitting due to vertical spacing.

This is a solvable problem. We could for example loop through the blocks twice.